src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -0,0 +1,206 @@
+import {
+  Box,
+  Button,
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  FormHelperText,
+  ListItemIcon,
+  ListItemText,
+  MenuItem,
+  Radio,
+  RadioGroup,
+  TableCell,
+  TableRow,
+  TextField,
+  Tooltip,
+  Typography,
+} from '@mui/material';
+import CancelIcon from '@mui/icons-material/Cancel';
+import prettyBytes from 'pretty-bytes';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { AssociatedFile, LicenseNames, licenses } from '../../../types/associatedArtifact.types';
+import { ConfirmDialog } from '../../../components/ConfirmDialog';
+import { Field, FieldProps, ErrorMessage, useFormikContext } from 'formik';
+import { SpecificFileFieldNames } from '../../../types/publicationFieldNames';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { DatePicker } from '@mui/x-date-pickers';
+
+interface FilesTableProps {
+  file: AssociatedFile;
+  removeFile: () => void;
+  toggleLicenseModal: () => void;
+  baseFieldName: string;
+}
+
+export const FilesTableRow = ({ file, removeFile, toggleLicenseModal, baseFieldName }: FilesTableProps) => {
+  const { t } = useTranslation();
+  const [openConfirmDialog, setOpenConfirmDialog] = useState(false);
+  const toggleOpenConfirmDialog = () => setOpenConfirmDialog(!openConfirmDialog);
+  const { setFieldValue, setFieldTouched } = useFormikContext();
+
+  return (
+    <TableRow key={file.identifier}>
+      <TableCell>{file.name}</TableCell>
+      <TableCell align="center">
+        <Tooltip title={t('common.delete')}>
+          <Button color="error" startIcon={<CancelIcon />} onClick={toggleOpenConfirmDialog} />
+        </Tooltip>
```

Comment:
Når man har en knapp som bare skal ha et ikon i seg kan man bruke MUI sin `IconButton` som er ment for akkurat dette. Da blir knappen tilpasset størrelsen til ikonet og litt sånn
```suggestion
        <IconButton title={t("registration.files_and_license.remove_file"}><CancelIcon/></IconButton>
```